server/models/volunteer.js

```diff
@@ -80,6 +84,36 @@ var VolunteerSchema = new Schema({
 VolunteerSchema.path('fields')
   .validate(getValidator(questionnaireIdentifiers.VOLUNTEER), 'Invalid field')
 
+  // Initialize geocoder options for pre save method
+  const geocoder = nodeGeocoder({
+    provider: 'google',
+    formatter: null
+  })
+
+  /**
+   * Hook a pre save method to construct the geolocation of the address
+   */
+  VolunteerSchema.pre('save', async function(next) {
```

Comment:
This could be extracted into a separate file because it's used in both volunteer and customer models